client/asset/eth/eth.go

```diff
@@ -780,7 +780,7 @@ func (eth *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin
 		copy(secretHash[:], swap.SecretHash)
 		receipts = append(receipts,
 			&swapReceipt{
-				expiration:   encode.UnixTimeMilli(int64(swap.LockTime)),
+				expiration:   time.Unix(int64(swap.LockTime), 0),
```

Comment:
Wow, good catch @JoeGruffins. Yikes.  I can confirm the other assets like DCR are indeed treating this `Contract.LockTime` field as seconds.   Also that `(*Core).swapMatchGroup` is preparing it in **seconds** like `matchTime.Add(t.lockTimeTaker).UTC().Unix()`.

So, righteous fix!  Only thing I think we should change is the docs on the `client/asset.Contract.LockTime` field so that it states the units.

Reply:
> Also, looks to have been broken in November, so it was once correct anyway.

Damn. That was me.

Reply:
is "UNIX seconds" correct English